integration/tests/cook/test_basic.py

```diff
@@ -661,20 +644,15 @@ def test_multiple_progress_updates_submit_stdout(self):
         job = util.wait_for_job(self.cook_url, job_uuid, 'completed')
         message = json.dumps(job, sort_keys=True)
         self.assertEqual('success', job['state'], message)
-
-        instance = util.wait_for_sandbox_directory(self.cook_url, job_uuid, 'success')
-        message = json.dumps(instance, sort_keys=True)
-        self.assertIsNotNone(instance['output_url'], message)
-        self.assertIsNotNone(instance['sandbox_directory'], message)
-        self.assertEqual('cook', instance['executor'])
-        util.sleep_for_publish_interval(self.cook_url)
```

Comment:
Is the exit code going to be populated after all the progress reporting has completed? If not, we might still need the sleep.

Reply:
Same as above.